### ISSUE TITLE
panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by shaimo** [celo-org/celo-blockchain#724]
My validator has been running for a couple of days while I was waiting to setup a proxy. Now just set it up and restarted the validator. Crashed as soon as it started to sync. Last few messages:
```
INFO [12-08|17:50:27.409] Commit new mining work                   number=10858 sealhash=9a3ceb…366f01 uncles=0 txs=0 gas=0 fees=0 elapsed=9.622ms
WARN [12-08|17:50:27.410] Block sealing failed                     err=unauthorized
ERROR[12-08|17:50:35.402] Failed to decode message from payload    address=0x186F77095e380F2d2F724E40e4352E67b0Fb8d42 func=handleMsg cur_seq=10858 cur_round=0 state="Waiting for new round" address=YYYYYYYYY err="unauthorized address"
INFO [12-08|17:50:35.485] Block synchronisation started 
INFO [12-08|17:50:35.486] Mining aborted due to sync 
WARN [12-08|17:50:35.487] Synchronisation failed, retrying         err="block download canceled (requested)"
INFO [12-08|17:50:35.506] Commit new mining work                   number=10858 sealhash=ee1154…87522a uncles=0 txs=0 gas=0 fees=0 elapsed=13.504ms
WARN [12-08|17:50:35.508] Block sealing failed                     err=unauthorized
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0xb7666e]

goroutine 137 [running]:
github.com/ethereum/go-ethereum/consensus/istanbul/backend.(*Backend).SendDelegateSignMsgToProxy(0xc0000fa240, 0xc02a912380, 0x301, 0x380, 0x380, 0x0)
        /go-ethereum/build/_workspace/src/github.com/ethereum/go-ethereum/consensus/istanbul/backend/backend.go:210 +0x1ae
github.com/ethereum/go-ethereum/ethstats.(*Service).handleDelegateSign(0xc0002a3200, 0xc02a7603c0, 0x0, 0x2)
        /go-ethereum/build/_workspace/src/github.com/ethereum/go-ethereum/ethstats/ethstats.go:474 +0x121
github.com/ethereum/go-ethereum/ethstats.(*Service).loop(0xc0002a3200)
        /go-ethereum/build/_workspace/src/github.com/ethereum/go-ethereum/ethstats/ethstats.go:290 +0x3c4
created by github.com/ethereum/go-ethereum/ethstats.(*Service).Start
        /go-ethereum/build/_workspace/src/github.com/ethereum/go-ethereum/ethstats/ethstats.go:160 +0x56
ro
```

**Comment by daithi-coombes** [comment]
possible duplicate: https://github.com/celo-org/celo-blockchain/issues/723

@shaimo seems the network is down at time of writing. Hopefully these issues disappear once its up again

**Comment by tkporter** [comment]
Should be fixed with the phase 1.1 reset that happened December 11 that includes this change: https://github.com/celo-org/celo-blockchain/pull/733/files#diff-6e58dd6edf43343da874fb58a00f6727R199
